content/source/assets/stylesheets/_variables.scss

```diff
@@ -25,7 +25,7 @@ $font-weight-reg:       400;
 $font-weight-bold:      600;
 
 // Body
-$body-font-color: $gray-darker;
+$body-font-color: #333;
```

Comment:
Peeking at the rest of the CSS, it looks like the only other place we use `$gray-darker` is the border of the "Feature Cards" on the home page:

https://github.com/hashicorp/terraform-website/blob/f4d46f7989a92efa5de2becf0bea917f411c7cdc/content/source/assets/stylesheets/_home.scss#L170-L172

![](https://user-images.githubusercontent.com/20180/55762850-6bc0f200-5a19-11e9-8fb6-f7c2436c8531.png)

I assume this matching of the text color inside the boxes with the border color was intentional. With that said, perhaps we should just change `$gray-darker` to be `#333` and let these both get darker together.

![](https://user-images.githubusercontent.com/20180/55762936-e38f1c80-5a19-11e9-8c13-0930809446bb.png)

Reply:
Looks like we also use it for `$home-header-link-color`, but yeah, I think probably that should be darker too!